classes/Configuration.php

```diff
@@ -366,10 +366,10 @@ public static function set($key, $values, $idShopGroup = null, $idShop = null)
         }
 
         if ($idShop === null) {
-            $idShop = Shop::getContextShopID(true);
+            $idShop = (int)Shop::getContextShopID(true);
```

Comment:
isn't worthier to cast directly in Shop::getContextShopId() function?

Reply:
not 100% sure of this, especially with an arg named "$null_value_without_multishop" :/

Reply:
Missing space after `(int)` :)

Reply:
I guess we have found someone motivated to work on our PHP CS Fixer configuration! :angel:

Reply:
@mickaelandrieu Casting in int convert null into int.

```bash
php -R '
$a = null;
var_dump((int) $a);
'

Command line run code:3:
int(0)
```

Reply:
ahhhhh ... php... :)